perses/rjmc/geometry.py

```diff
@@ -551,8 +540,11 @@ def _torsion_potential(self, torsion, phi, beta):
         gamma = torsion.type.phase
```

Comment:
Also, you have `phi = phi/phi.unit` above---probably not a good idea!  Can you attach proper units to `phi`, `beta`, `phase`, and `V` and then make sure you multiply them together into a non-unit-bearing quantity?